HW_5/task_3.py

```diff
@@ -0,0 +1,11 @@
+data = {'name1': 100, 'name2': 200, 'name3': 300}
+bonus = ['10.25%', '5%', '7.5%']
+result = {k: v * (1 + float(bonus[i].strip('%')) / 100)
+          for i, (k, v) in enumerate(data.items())}
+
+names = ['Павел', 'Мария', 'Максим']
+wages = [15000, 25000, 20000]
+bonuses = ['10.25%', '6%', '7.5%']
+result = {k: v * (1 + float(bonuses[i].strip('%')) / 100)
+          for i, (k, v) in enumerate(zip(names, wages))}
```

Comment:
А почему так? Когда можно так
```suggestion
result = {n: w * (1 + float(b.strip('%')) / 100) for n, w, b in zip(names, wages, bonuses)}
```